src/extension.ts

```diff
@@ -368,7 +372,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 }
 
 export async function deactivate(): Promise<void> {
-    vscode.window.showInformationMessage('The extension "vscode-makefile-tools" is de-activated');
+    vscode.window.showInformationMessage(localize("extension.deactivated", "The extension 'vscode-makefile-tools' is de-activated."));
```

Comment:
You may want to move the extension name (anything you don't want to see translated) into a variable that gets provided via a `{0}` placeholer.